### DISCUSSION TITLE
Build statistics layer into gEAR

### DISCUSSION THREAD

**Discussion by jorvis** [IGS/gEAR#242] [Ideas]
This will be a parent/discussion ticket regarding creating much more advanced tracking and reporting of gEAR usage than Google Analytics can provide.  Feel free to edit/add to any of these and I'll create subtickets with effort estimates.

**Comment by jorvis** [comment]
Make sure all entity tables have ldesc and date_added fields

**Comment by adkinsrs** [comment]
Linking to #1236
